### DISCUSSION TITLE
tai闪退问题

### DISCUSSION THREAD

**Discussion by freecube0209** [Planshit/Tai#238] [Q&A] [answered]
系统win10，tai版本1.5.0.1，偶尔出现tai自动闪退，手动重启以后仍然不一会就闪退的情况。已打开网页统计功能，杀毒软件已添加信任。电脑配置充足。

**Comment by ghost** [comment]
我这也会闪退，都不知道什么时候死的

**Comment by freecube0209** [comment, marked as answer]
开发者已邮件回复，确认为网页统计功能导致，建议暂时关闭。希望早日解决。
